src/main/webapp/index.html

```diff
@@ -39,7 +39,8 @@ <h1>Pieced Homepage</h1>
 
         <!-- Access to test JSON file; remove later -->
         <script src="test.json"></script>
-        <script src="species.js"></script>
+    <script src="scripts/pixelation.js"></script>
+    <script src="scripts/fetchData.js"></script>
```

Comment:
I'm not sure if this is just github, but the indentation of these two don't match the rest

Reply:
Oops missed that, thanks